conda-forge.yml

```diff
@@ -18,6 +18,8 @@ os_version:
   linux_64: cos7
 provider:
   linux_aarch64: default
+  linux_ppc64le: default
+  # Note: linux_ppc64le is currently build manually by @matthiasdiener
```

Comment:
I think this is not a good place to place this comment because it could be overwritten by the conda-forge bots. 

Instead, I'd suggest to add yourself to the list of recipe maintainers and add this note next to your Github handle.